test/perf/shootout/k_nucleotide.jl

```diff
@@ -46,7 +46,7 @@ function sorted_array(m::Dict{AbstractString, Int})
         kn[i] = KNuc(elem...)
         i += 1
     end
-    sort!(kn)
+    sort(kn)
```

Comment:
I have to say, I don't understand why using `sort!` here is not ok. We use `f.()` in the Laplace benchmark now, how is using inplace APIs where available any different than that?